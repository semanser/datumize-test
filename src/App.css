.App {
	background: #F4F4F4;
  height: 100%;
}

.page-title {
	color: #B7B7B7;
  font-size: 1.3em;
  font-weight: 400;
  padding: 40px 0;
  text-align: center;
}

.widget {
	-moz-box-shadow: 0px 2px 4px 0px rgba(207,207,207,1);
	-webkit-box-shadow: 0px 2px 4px 0px rgba(207,207,207,1);
	background: #FFFFFF;
	box-shadow: 0px 2px 4px 0px rgba(207,207,207,1);
  border-radius: 3px;
  margin: 0 auto;
  max-height: 300px;
  overflow: scroll;
  padding: 10px;
  width: 260px;
  position: relative;
  min-height: 200px;
}

.widget .Select-control {
  border: none;
	border-bottom: 1px solid #EDEDED;
  font-size: 18px;
  outline: none;
  padding-bottom: 6px;
  margin-bottom: 8px;
}

.widget .Select-control:hover {
  box-shadow: none;
}

.widget .Select-value-label {
	color: #898989 !important;
  font-weight: 500;
}

.roles-item {
	color: #888888;
  font-size: 13px;
  font-weight: 300;
  height: 35px;
  margin: 0px -10px;
  position: relative;
  cursor: default;
}

.roles-item:hover {
  background: #FAFAFA;
}

.roles-item .icon {
  border-radius: 50%;
  border: 2px solid #fff;
  color: #fff;
  float: left;
  font-size: 10px;
  height: 30px;
  margin-left: 18px;
  margin-right: 10px;
  margin-top: 2px;
  padding-top: 7px;
  text-align: center;
  width: 30px;
}

.roles-item .text {
  padding: 9px 0;
}

.roles-item .role {
	color: #ACACAC;
  cursor: pointer;
  font-size: 12px;
  position: absolute;
  right: 18px;
  top: 10px;
}

.roles-item .role:hover {
  text-decoration: underline;
}

.roles-item .role.unassign {
  font-weight: 400;
}

.widget .assign {
	color: #AEAEAE;
  background: #fff;
  border: none;
  bottom: -10px;
  cursor: pointer;
  display: block;
  height: 30px;
  position: sticky;
  width: 100%;
}

.widget .assign:hover {
	color: #989898;
}

.widget .header {
  padding: 9px;
}

.widget .header .title {
  font-size: 14px;
	color: #898989;
}

.widget .header .description {
	color: #B7B7B7;
  font-size: 12px;
  font-weight: 300;
}

.widget .loader > div{
  margin: 70px auto;
}
