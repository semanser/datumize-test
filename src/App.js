import React, { Component } from 'react'
import './App.css'
import RolesWidget from './containers/RolesWidget'

class App extends Component {
  render() {
    return (
      <div className="App">
        <div className="page-title">
          Select project below to manage assignments
        </div>
        <RolesWidget />
      </div>
    )
  }
}

export default App
