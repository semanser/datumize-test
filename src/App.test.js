import React from 'react'
import ReactDOM from 'react-dom'
import ShallowRenderer from 'react-test-renderer/shallow'
import App from './App'

const renderer = new ShallowRenderer()

describe('App Component', () => {
  it('Renders component without crashing', () => {
    const div = document.createElement('div')
    const tree = renderer.render(<App />)
    expect(tree).toMatchSnapshot()
  })
})

