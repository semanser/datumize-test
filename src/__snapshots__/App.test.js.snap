// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component Renders component without crashing 1`] = `
<div
  className="App"
>
  <div
    className="page-title"
  >
    Select project below to manage assignments
  </div>
  <Connect(RolesWidget) />
</div>
`;
